recipes/pytest-datadir/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-datadir" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "898020f9865c56844399f28475e0c7ef8c7ca58614929fd07cbb30ee2d8e2657" %}
+{% set ext = "tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ ext }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pytest_datadir
+  commands:
+    - py.test --help
+
+about:
+  home: https://github.com/gabrielcnr/pytest-datadir
+  license: LGPL
```

Comment:
[Here](https://github.com/gabrielcnr/pytest-datadir/blob/master/setup.py#L30) is says MIT. But the previous release seems to be `LGPL-3.0`.